azure/azure-how-to/instances/find-ubuntu-images.rst

```diff
@@ -213,6 +219,22 @@ Ubuntu CVM 22.04 LTS - Jammy Jellyfish
      - Gen2
      - ``Canonical:0001-com-ubuntu-confidential-vm-jammy:22_04-lts-cvm:latest``
 
+Ubuntu Pro CVM 22.04 LTS - Jammy Jellyfish
```

Comment:
Do we really need this separate section? Couldn't we just add this one to the table above (in the one for 'Ubuntu CVM 22.04 LTS - Jammy Jellyfish') - like you did for Noble?

Reply:
Unlike the Noble, the Jammy Pro CVM offer is a standalone offer. It cannot be sensibly grouped in under the `Ubuntu CVM 22.04 LTS - Jammy Jellyfish` section as it belongs to a completely separate offer with its own links (and as an Azure Virtual Machine is not available in China, so the quickstart link there does not apply).

This is another issue that has been raised with Alliances. Realistically we should not have even produced Pro CVM for Jammy, but we agreed to proceed with it out of Madrid so we're left with another edge case to work around.